.github/workflows/github_pages.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - releases/v1.0.0
```

Comment:
please remove "develop" target branch from the list.

Reply:
I see.